<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>421</width>
      <height>290</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="2">
        <widget class="QDial" name="dial_7">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Homework 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Homework 1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_2">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Homework 7</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Homework 3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDial" name="dial_4">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDial" name="dial_6">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_3">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBox_4">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Homework 4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDial" name="dial_3">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Homework 8</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Homework 5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Homework 6</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="spinBox_5">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBox_6">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="spinBox_7">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QSpinBox" name="spinBox_8">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDial" name="dial_2">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QDial" name="dial_8">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QDial" name="dial_5">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDial" name="dial">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>101</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_midterm1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Midterm 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="dial_midterm1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_midterm1">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_midterm2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Midterm 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="dial_midterm2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_midterm2">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Final Exam</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="dial_final">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_final">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrade_Calculator">
    <property name="title">
     <string>Grade Calculator</string>
    </property>
   </widget>
   <addaction name="menuGrade_Calculator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
